server/runtime/src/main/java/org/infinispan/server/Server.java

```diff
@@ -434,6 +436,11 @@ public synchronized CompletableFuture<ExitStatus> run() {
          // Register ourselves with the global registry
          GlobalComponentRegistry gcr = SecurityActions.getGlobalComponentRegistry(cacheManager);
          gcr.registerComponent(this, ServerManagement.class);
+
+         if (gcr.getGlobalConfiguration().tracing().security()) {
```

Comment:
I think this one won't change in runtime.

Reply:
You're right. Security attribute is a global level configuration attribute, so it cannot be changed ad runtime.